dbt_style_guide.md

```diff
@@ -531,111 +531,156 @@ when needed.
   ) %}
   ```
 
-## Metrics style guide
+## Semantic Models & Metrics style guide
+
+## Organizing Semantic Model
+
+- Semantic models are created in the same yml file where you define configs, tests and metrics for your models. There should be one yml file per model.
+- This keeps all the semantic models in your project in one place, so you don't have to hunt for them during development.
+
+```bash
+├── dbt_project.yml
+└── models
+    ├── marts
+				├── finance
+					  ├── orders.sql
+						├── orders.yml #semantic models defined here
+    ├── staging
+			  ├── product
+				    ├── products.sql
+						├── products.yml #semantic models defined here

```

Comment:
Similar to the [Python style guide for comments in PEP-8](https://peps.python.org/pep-0008/#inline-comments), inline comments in YAML should be separated by at least two spaces from the statement. They should start with a # and a single space.

```suggestion
						├── orders.yml  # semantic models defined here
    ├── staging
			  ├── product
				    ├── products.sql
						├── products.yml  # semantic models defined here

```